src/app/ui/bridge/StepDetail.tsx

```diff
@@ -0,0 +1,56 @@
+'use client';
+import { Box, Typography } from '@mui/material';
+import Image from 'next/image';
+import type { ReactElement } from 'react';
+import { StepDetailContainer } from './StepDetail.style';
+
+const WIDGET_AMOUNT = 3;
+
+interface StepDetailImageProps {
+  imgUrl: string;
+  width: number;
+  height: number;
+  alt: string;
+}
+
+interface StepDetailProps {
+  title: string;
+  description: string;
+  img: StepDetailImageProps;
+  content?: ReactElement;
+}
+
+const StepDetail = ({ title, description, img, content }: StepDetailProps) => {
+  return (
+    <StepDetailContainer
+      sx={(theme) => ({ marginTop: theme.spacing(4) })}
+      imgWidth={img.width}
+    >
+      <Box>
+        <Typography
+          variant="h4"
+          marginY={2}
+          sx={{ fontSize: '24px!important' }}
+        >
+          {title}
+        </Typography>
+        <Typography>{description}</Typography>
+        {content ? content : null}
```

Comment:
this statement is not needed

Reply:
Thanks, good point

Reply:
Was resolved 4 days ago:
https://github.com/jumperexchange/jumper-exchange/pull/1557/commits/e0ea34e4a495d44d5308acd202806a2b1a351c9e